src/Presentation/Components/NavigationBar.tsx

```diff
@@ -36,22 +41,22 @@ function NavigationBar() {
 
     return (
         <>
-            <Navbar bg="dark" variant="dark">
+            <Navbar className='nav-background'>
                 <Container>
-                    <Navbar.Brand><Link to='/'>Dodo</Link></Navbar.Brand>
+                    <Navbar.Brand><Link to='/' className='navbar-brand'><img src={logo} width="45" height="45" alt="Logo image of a smiling dodo bird." /> Dodo</Link></Navbar.Brand>
                     <Nav>
-                        <Link to='/'> <HomeIcon /> Home</Link>
-
                         { auth.currentUser === null ?
                             <>
-                                <Link to='/login'>Login</Link>
-                                <Link to='/registration'>Registration</Link>
+                                <Link to='/login' className='nav-link'><LoginIcon className='icon-color'/> Login</Link>
+                                <Link to='/registration' className='nav-link'><CreateIcon className='icon-color'/> Registration</Link>
                             </> :
                             <>
-                                <Link to='/create-flashcards'> <LibraryAddIcon className='card-add'/> Create </Link>
+                                <Link to='/create-flashcards' className='nav-link'><LibraryAddIcon className='icon-color'/> Create</Link>
                                 <Link to='/view-flashcards'> <StyleIcon /> Cards</Link>
```

Comment:
Navbar is looking great, but I think if you add `className='nav-link'` to the cards link, it will get rid of the underline.
![image](https://user-images.githubusercontent.com/65355965/205787746-c07b4fec-255c-4a23-bd8e-0d66dd73585d.png)